en/user-guide/track_lines_polygons.rst

```diff
@@ -0,0 +1,51 @@
+.. _track_linges_polygons:
+
+Tracking of lines and polygons
+==============================
+
+You have the possibility to do tracking on lines and polygons.
+
+The option is available when GPS is enabled. 
```

Comment:
```suggestion
It's available when GPS is enabled. 
```

Reply:
"It is" <- I think in "professional writing" the long form is preferred.

Reply:
Are you calling me an un-professional writer? :stuck_out_tongue_closed_eyes:
I am fine with "It is"...